tests/func/test_import.py

```diff
@@ -266,6 +270,21 @@ def test_pull_wildcard_imported_directory_stage(tmp_dir, dvc, erepo_dir):
     assert (tmp_dir / "dir_imported123").read_text() == {"foo": "foo content"}
 
 
+def test_push_pull_glob_no_match(
```

Comment:
Probably this test makes more sense to be located in `tests/func/test_data_cloud.py`.

Reply:
Thanks, makes sense. Moved to test_data_cloud.py